checkout_sdk/payments/payments_apm.py

```diff
@@ -122,6 +190,31 @@ def __init__(self):
         super().__init__(PaymentSourceType.PAGOFACIL)
 
 
+class RequestPayPalSource(RequestSource):
+    invoice_number: str
+    recipient_name: str
+    logo_url: str
+    stc: dict
+
+    def __init__(self):
+        super().__init__(PaymentSourceType.PAYPAL)
+
+
+class RequestPoliSource(RequestSource):
+    def __init__(self):
+        super().__init__(PaymentSourceType.POLI)
+
+
+class RequestQPaySource(RequestSource):
+    quantity: str
```

Comment:
This is listed as an integer in the docs. Is it ok to have it as a string here?

Reply:
Done, good call out Martin, Thanks!